code/game/machinery/constructable_frame.dm

```diff
@@ -410,7 +410,7 @@ obj/item/weapon/circuitboard/rdserver
 	
 /obj/item/weapon/circuitboard/chem_dispenser/booze_dispenser
 	name = "Circuit Board (Booze Dispenser)"
-	build_path = " /obj/machinery/chem_dispenser/booze_dispenser"
+	build_path = "/obj/machinery/chem_dispenser/booze_dispenser"

```

Comment:
There's some old world reason behind it, something to do with being able to edit a string but not a path in VV.